c2corg_ui/templates/route/view.html

```diff
@@ -4,8 +4,13 @@
 <%inherit file="../base.html"/>
 <%namespace file="../helpers/view.html" import="show_attr, show_short_attr, show_missing_langs_links,
     show_other_langs_links, show_archive_data, show_route_title, show_associated_waypoints,
-    show_associated_routes, show_areas, show_maps"/>
+    show_associated_routes, show_areas, show_float_buttons, show_maps"
+ />
+<%namespace file="detailedattributes.html" import="get_route_gear, get_route_location, get_route_rating, get_route_general, 
```

Comment:
Since the file only contains functions (helpers), what about making the filename more explicit, including "helpers" somewhere or even by creating an ``helpers/`` directory?